spec/miscellaneous/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -46,3 +46,49 @@
     end
   end
 end
+
+describe "rake After:unhide_invited_works" do
+  let(:invited_anonymous_collection) { create(:anonymous_collection) }
+  let(:invited_unrevealed_collection) { create(:unrevealed_collection) }
+  let(:anonymous_collection) { create(:anonymous_collection) }
+  let(:unrevealed_collection) { create(:unrevealed_collection) }
+  let(:other_collection) { create(:collection) }
+  let(:invited_anonymous_unrevealed_collection) { create(:anonymous_unrevealed_collection) }
+  let(:invited_anonymous_work) { create(:posted_work, collections: [invited_anonymous_collection]) }
```

Comment:
I think its simpler and clearer to have one work per collection as it makes the breaking the collection item deterministic and not dependent of the order of the items in the collection.

Reply:
It wouldn't depend on the order of the works in the collection. You're finding the `collection_item` based on the work: `invited_anonymous_work.collection_items.first.update_columns(user_approval_status: CollectionItem::NEUTRAL)`.  Each work will still only have one `collection_item` because it will only be in one collection.

Reply:
Your right.